terraform/cross-device/storage.tf

```diff
@@ -17,7 +17,7 @@ module "buckets" {
   version    = "5.0.0"
   project_id = data.google_project.project.project_id
   location   = var.region
-  prefix     = "fcp-${var.environment}"
+  prefix     = "fcp-${var.environment}-${random_id.bucket_prefix.hex}"
```

Comment:
You don't need to do this yourself. Check the cloud-storage module docs. It supports adding a randomized suffix.

https://registry.terraform.io/modules/terraform-google-modules/cloud-storage/google/latest

Reply:
Done